tests/foreman/cli/test_reporttemplates.py

```diff
@@ -745,6 +745,7 @@ def test_positive_generate_ansible_template(module_target_sat):
     assert host['name'] in [item.split(',')[1] for item in report_data.split('\n') if len(item) > 0]
 
 
+@pytest.mark.no_containers
 @pytest.mark.tier3
 def test_positive_generate_entitlements_report_multiple_formats(
     module_entitlement_manifest_org, local_ak, local_subscription, rhel7_contenthost, target_sat
```

Comment:
Also, it's probably best to switch to `module_sca_manifest_org` while at it. I can see the test is failing in stream because of using `module_entitlement_manifest_org`, or specifically when trying to disable sca.
```suggestion
    module_sca_manifest_org, local_ak, local_subscription, rhel7_contenthost, target_sat
```
It's also being used by `setup_content` so you'll have to update that too.